example/push_notifications/pubspec.yaml

```diff
@@ -33,8 +33,6 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.6
-  firebase_core: ^2.20.0
-  firebase_messaging: ^14.7.2
```

Comment:
Now, I know that `firebase_core` and `firebase_messaging` are automatically installed as they are dependencies to `talkjs_flutter`, but isn't it more explicit to require them also in the example app, given that in order for push notifications to work, we need to initialize Firebase in our main function?

Reply:
But isn't that why the SDK already depends on them. If the user is using Firebase for their own purposes, then I do agree that having it explicitly is necessary however if they are using Firebase only for purposes of push notifications from TalkJS like in the above case, then relying on it transitively is fine.

Reply:
I don't hold any strong opinions on this. I just felt having them as direct dependencies wasn't necessary.

Reply:
Still, to me it feels strange to call

```dart
  await Firebase.initializeApp(
    options: DefaultFirebaseOptions.currentPlatform,
  );
```

and not having firebase as a direct dependency